nx.json

```diff
@@ -22,24 +22,6 @@
         "^build"
       ]
     },
-    "release": {
-      "dependsOn": [
-        "build",
-        "^build"
-      ]
-    },
```

Comment:
Renamed this command from `release` to `publish` a while ago, this was a leftover